website/src/Layout.tsx

```diff
@@ -784,7 +772,26 @@ function MDXContent({ toc: headings, children }: ComponentPropsWithoutRef<Nextra
           >
             {children}
           </section>
-          <div className="col-[container] flex justify-end pb-12 group-data-[hide-table-of-contents]/layout-toc-grid:hidden xl:hidden">
```

Comment:
Did you intentionally remove the `group-data-[hide-table-of-contents]/layout-toc-grid:hidden` class?

Reply:
Ah, sorry, I misunderstood what this was. This is fine :)